packages/babel-preset/src/types.ts

```diff
@@ -26,3 +31,13 @@ export type BabelPluginOptions = {
    */
   projectRoot?: string;
 };
+
+type CSSRulesBySlot = Record<string, string[]>;
+
+export type GriffelHookEntries = Record<string, CSSRulesBySlot>;
+export type GriffelResetHookEntries = Record<string, string[]>;
```

Comment:
I am not a big fan of re-exporting this, can we inline them under `BabelPluginMetadata`?